lib/dokumi/tool/android.rb

```diff
@@ -26,6 +26,20 @@ def findbugs(target_project)
         end
       end
 
+      def infer(target_project)
+        @environment.action_executed = true
+
+        Support::Shell.run("infer", "--", "./gradlew", "build")
```

Comment:
I talked with slightair and understood that it is necessary to execute gradle because there is no `local.properties`.